src/store/userDataStore.ts

```diff
@@ -0,0 +1,43 @@
+import electron from "electron";
+import * as fs from "fs";
+import * as path from "path";
+
+export class UserDataStore {
+    private path: string;
+    private data: object;
+
+    constructor() {
+        // Renderer process has to get `app` module via `remote`, whereas the main process can get it directly
+        // app.getPath('userData') will return a string of the user's app data directory path.
+        const userDataPath = (electron.app || electron.remote.app).getPath("userData");
```

Comment:
Do we know for sure this works on linux/mac/windows? userData seems something from windows only.

Reply:
This works on Linux (probably also Windows, will test later), because I work on Ubuntu. I'm not sure about macOS. As far as I know, this function returns a different path based on the system the application runs on.